packages/nameguard-react/src/components/Report/CheckResultCard.tsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
-import { CheckResult } from "@namehash/nameguard";
+import { CheckResult, CheckResultCode } from "@namehash/nameguard";
 import { CheckResultCodeIcon } from "./CheckResultCodeIcon";
 
 export function CheckResultCard(props: CheckResult) {
+  const status = props ? props.status : CheckResultCode.pass;
```

Comment:
This suggests our type system is being violated. It should be impossible for `props` not not be defined when this function is called. The param is not identified as being optional and I don't believe it should be optional. It should be the responsibility of the caller of this function to handle the possibility that they don't have a `CheckResult` and to manually pass a value such as `CheckResultCode.pass` as needed.